modules/creating-control-plane-node.adoc

```diff
@@ -0,0 +1,387 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machines_management/cpmso-manually-scaling-control-planes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-control-plane-node_{context}"]
+= Adding a control plane node to your cluster
+
+You can add 4 or 5 control plane nodes to the `ControlPlaneMachineSet` object that exists in your cluster. You can then verify the healthiness of the nodes. 
```

Comment:
You can only have 4 or 5 node control planes on clusters without ControlPlaneMachineSet, and IIRC, we restrict this only to baremetal clusters as well. We shouldn't assume that this is valid for platforms like AWS, Azure, GCP etc